shell_test.go

```diff
@@ -12,6 +12,14 @@ func TestBashEscape(t *testing.T) {
 	assertEqual(t, `$'\xc3\xa9'`, BashEscape("é"))
 }
 
+func TestElvishEscape(t *testing.T) {
+	escape := (elvish{}).escape
+	assertEqual(t, ``, escape(""))
+	assertEqual(t, `escape'quote`, escape("escape'quote"))
+	assertEqual(t, `foo bar`, escape("foo bar"))
+	assertEqual(t, `é`, escape("é"))
```

Comment:
Can you add a test for `"` and `\` as well ?

Reply:
done.